.github/workflows/ci-run-test.yml

```diff
@@ -373,9 +373,10 @@ jobs:
         shell: bash
         run: mkdir status; echo "OK" > "${{ env.TEST_STATUS_FILE }}"
       - name: Register test successful.
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
           path: ${{ env.TEST_STATUS_FILE }}
+          overwrite: true
```

Comment:
I think here we are making a later write to the test status file to indicate success, which means it's OK to overwrite the previous instance?

Reply:
Compute says no

😭